streampipes-client-python/mkdocs.yml

```diff
@@ -106,3 +106,8 @@ nav:
       - Exploring Live Data from a StreamPipes Data Stream: tutorials/3-getting-live-data-from-the-streampipes-data-stream.ipynb
       - Applying Online Machine Learning on a StreamPipes Data Stream: tutorials/4-using-online-machine-learning-on-a-streampipes-data-stream.ipynb
   - 📚 Reference: reference/*
+  theme:
+  name: material
```

Comment:
This is a duplicate of line 29

Reply:
should i remove this or replace it

Reply:
You can remove it, since it doesn't add any value

Reply:
already done thanks for feedback

Reply:
Have you updated the PR? It's still contained in the diff
Same applies for the other aspects I've put a comment on